### PR TITLE
update wrapper examples

### PR DESCRIPTION
**Pull request by dmlittle**
What:
 - add CONTRIBUTING.md
 - update copyright year
 - add bulk CSV for Postcards and Letters
 - remove unused imports

Why:
 - general update & cleanup of the repo

### CODE REVIEW

File: lob-java-examples/src/main/java/com/lob/examples/CsvLetterExample.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            final CsvLetter letter = iterator.next();
+
+            final Map<String, String> data = new HashMap<String, String>();
+            data.put("date", today);
+            data.put("name", letter.name);
+            data.put("amountDue", letter.amount);
+
+            final LetterRequest letterRequest = LetterRequest.builder()
+                    .to(
+                            AddressRequest.builder()
```
Comment (mgartner) [inline lob-java-examples/src/main/java/com/lob/examples/CsvLetterExample.java]: Is this formatting standard?

**Comment by mgartner** [comment]
LGTM

File: lob-java-examples/src/main/java/com/lob/examples/CsvLetterExample.java (hunk 1 above)
Comment (dmlittle) [inline lob-java-examples/src/main/java/com/lob/examples/CsvLetterExample.java]: There is no standard convention for line-wrapping but it's standard to increase the indentation space for a new block. Although this isn't a new block, it is all supposed to be enclosed in the `.to()`. For what it's worth, it's the convention in the JUnit tests.

https://github.com/lob/lob-java/blob/9cdeb6d15066d7faaea9d6eeaf57635d601ddeb2/src/test/java/com/lob/client/test/PostcardTest.java#L109

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# How to contribute
+We welcome contributions from the community and are pleased to have them.  Please follow this guide when logging issues or making code changes.
+
+## Logging Issues
+All issues should be created using the [new issue form](https://github.com/lob/lob-java/issues/new).  Clearly describe the issue including steps to reproduce if there are any.  Also, make sure to indicate the version of the wrapper that has the issue being reported.
```
Comment (elnaz) [inline CONTRIBUTING.md]: nit: extra space before `Clearly`.

Comment (elnaz) [inline CONTRIBUTING.md]: I think we can end on `indicate the version of the wrapper`.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+## Patching Code
+Code changes are welcome and should follow the guidelines below.
+
+* Fork the repository on GitHub.
+* Add tests for your new code ensuring that you have 100% code coverage (we can help you reach 100% but will not merge without it).
+    * Run `mvn test` to make sure all tests are still passing. You can see the code coverage report under `target/site/jacoco/index.html`
+* [Pull requests](http://help.github.com/send-pull-requests/) should be made to the [master branch](https://github.com/lob/lob-java/tree/master).
```
Comment (elnaz) [inline CONTRIBUTING.md]: Should we also mention to run the examples and make sure they're still working?

**Comment by elnaz** [comment]
This is great Donald! Could you also add a readme in the examples directory similar to what we have for [other](https://github.com/lob/lob-python/tree/master/examples) [wrappers](https://github.com/lob/lob-node/tree/master/examples)?

**Comment by dmlittle** [comment]
@elnaz done! Can you make sure that the command-line examples work? I want to make sure I didn't also do something else locally to make it work that I might have excluded.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+# How to contribute
+We welcome contributions from the community and are pleased to have them.  Please follow this guide when logging issues or making code changes.
+
+## Logging Issues
+All issues should be created using the [new issue form](https://github.com/lob/lob-java/issues/new). Clearly describe the issue including steps to reproduce if there are any.  Also, make sure to indicate the version of the wrapper.
```
Comment (elnaz) [inline CONTRIBUTING.md]: nit: remove the extra space before `Also`.

File: lob-java-examples/README.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+# Java Examples
+Here we have put together a hand full of java examples to help get you started. As always feel free to [contact us](https://lob.com/support) directly if you have any questions on implementation.
+
+
+## Getting started
+Before running these examples make sure you are in the `lob-java-examples/` directory.
+
+```bash
+lob$ cd lob-java-examples/
+lob-java-examples$ mvn clean package;
```
Comment (elnaz) [inline lob-java-examples/README.md]: Is the semicolon preferred? I was able to run it without it.

**Comment by elnaz** [comment]
These command line example instructions are great!

It looks like the CsvVerificationExample is [referencing an input.csv](https://github.com/lob/lob-java/pull/99/files#diff-abecd97b1e53f2d2eda7339fa5ebc6e6R20) that has moved or is missing.

File: lob-java-examples/src/main/java/com/lob/examples/CsvLetterExample.java [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+                    .build();
+
+            try {
+                final LetterResponse letterResponse = client.createLetter(letterRequest).get();
+                printResponse("Letter", letterResponse);
+            }
+            catch (final Exception e) {
+                printResponse("Failed Letter", letter);
+            }
+        }
```
Comment (elnaz) [inline lob-java-examples/src/main/java/com/lob/examples/CsvLetterExample.java]: This was where we added the System.exit

**Comment by elnaz** [comment]
Nice! My only nit is the `resources` folder because I got that confused with API resources. Maybe `assets` is more clear? Not a strong opinion either way.

LGTM 🎈

**Comment by dmlittle** [comment]
@elnaz I'll keep the `resources/` folder name as it's the default folder name that gets compiled together with the source code for assets.

**Pull request merged**